src/ui/public/field_editor/field_editor.html

```diff
@@ -226,7 +226,7 @@ <h4 class="hintbox-heading">
           <li> Comparison operators: < <= == >= > </li>
           <li> Common mathematic functions: abs ceil exp floor ln log10 logn max min sqrt pow </li>
           <li> Trigonometric library functions: acosh acos asinh asin atanh atan atan2 cosh cos sinh sin tanh tan </li>
-          <li> Distance functions: haversin </li>
+          <li> Distance functions: haversion </li>
```

Comment:
I'm going to bet that this is meant to be "haversine" (https://en.wikipedia.org/wiki/Haversine_formula).

@thomasneirynck WDYT?

Reply:
Oops, coincidentally damaged/highlighted.

My system does substring replacements so that was from `version`, happy to fix (probably tonight)

Reply:
Hmm, I was wrong, I actually intentionally picked that correction, not sure how. Your correction is of course better.